package.json

```diff
@@ -48,6 +48,7 @@
     "compression": "^1.7.3",
     "cookie-parser": "^1.4.3",
     "cookie-session": "^2.0.0-beta.3",
+    "date-fns": "^1.30.1",
```

Comment:
Was there a decision to use `date-fns` instead of moment for new projects? It's fine with me but I'd like to see the decision recorded and communicated if it hasn't been.

Reply:
We had a chat in Slack about a light(er)weight solution for dates, considering we need very little. Made a DR and tagged you in Notion.